cves/CVE-2019-9721.yml

```diff
@@ -306,8 +317,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: User file inputs should always be expected to be malicious.
```

Comment:
This is not well stated. User file inputs should be expected to be normal most of the time, but it's that one-in-a-million bad actor who will find and leverage a vulnerability like this.